client/extensions/woocommerce/app/dashboard/manage-no-orders.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import BasicWidget from '../../components/basic-widget';
+import ExtensionsWidget from '../../components/extensions-widget';
+import { getLink } from '../../lib/nav-utils';
+import ReadingWidget from '../../components/reading-widget';
+import ShareWidget from '../../components/share-widget';
+import WidgetGroup from '../../components/widget-group';
+
+class ManageNoOrders extends Component {
+	static propTypes = {
+		site: PropTypes.shape( {
+			slug: PropTypes.string.isRequired,
+		} ),
+	};
+
+	renderShareWidget = () => {
+		const { site, translate } = this.props;
+		return (
+			<ShareWidget
+				text={ translate( 'Now that your setup is complete it\'s time to spread the word' +
+					' and get those first orders rolling in. Share a link with your friends,' +
+					' family & followers now.' ) }
+				title={ translate( 'Your store is ready & the world is waiting!' ) }
+				urlToShare={ getLink( 'https://:site', site ) }
+			/>
+		);
+	}
+
+	renderReadingWidget = () => {
+		const { site, translate } = this.props;
+		return (
+			<ReadingWidget
+				className="dashboard__reading-widget"
+				site={ site }
+				text={ translate( 'Choice articles for new store owners to read. Get tips' +
+					' from seasoned merchants, learn best practices to keep your store' +
+					' ship-shape and prepare for success.' ) }
+				title={ translate( 'Recommended reading' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Recommended Reading' )` *ES Score: 22*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-dashboard-fledgling)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).